TurkishFilter/sections/specific.txt

```diff
@@ -221,8 +221,7 @@ manga-tr.com#?#.col-sm-4 > .well3:has(> .panel > .panel-heading2 > center:contai
 [$domain=lifebursa.com,path=/^\/$/]#?#.container > div.row:has(> div[class^="col-"] > div.banner)
 eksisozluk1923.com#$?##interstitial-close-link-source { remove: true; }
 eksisozluk1923.com#$?#.ad-doubleclickwebinterstital { remove: true; }
-eksisozluk1923.com#$?#.ui-autocomplete > li:has(> div[style="color:rgb(0, 0, 0)"]) { remove: true; }
-eksisozluk1923.com#$?#.ui-autocomplete > li[style^="position: absolute;"][style*="z-index:"]:has(> .ui-menu-item-wrapper:contains(/^sponsorlu$/)) { remove: true; }
+eksisozluk1923.com###search-box-ad
```

Comment:
Why the rules were removed? It's not enough on my end. I'll add the suggested one.